src/main/java/nextstep/courses/domain/session/CoverImage.java

```diff
@@ -0,0 +1,61 @@
+package nextstep.courses.domain.session;
+
+import static nextstep.courses.ExceptionMessage.INVALID_COVER_IMAGE;
+import static nextstep.courses.domain.session.ImageType.*;
+
+public class CoverImage {
+    private static final double MAX_CAPACITY = 1024 * 1024;
+    private static final double MIN_WIDTH = 300;
+    private static final double MIN_HEIGHT = 200;
+    private static final double RATIO_OF_WIDTH_HEIGHT = 3.0 / 2.0;
+
+    private final String name;
+    private final double capacity;
+    private final double width;
+    private final double height;
```

Comment:
```suggestion
    private final double width;
    private final double height;
```

객체로 관리된다면 책임을 줘 볼수 있을것 같아요 🤔 
인스턴스 변수도 줄일 수 있을것 같구요 😄